OpenMEEG/src/assemble.cpp

```diff
@@ -268,25 +268,49 @@ int main(int argc, char** argv)
     * (i.e. the potential and the normal current on all interfaces)
     * |----> v (potential at the ECoG electrodes)
     **********************************************************************************************/
-    else if ( option(argc, argv, {"-Head2ECoGMat", "-H2ECogM", "-H2ECOGM", "-h2ecogm"},
+    else if (argc==6 && option(argc, argv, {"-Head2ECoGMat", "-H2ECogM", "-H2ECOGM", "-h2ecogm"},
+                                            {"geometry file", "conductivity file", "ECoG electrodes positions file", "output file"})) {
+
+        // Loading surfaces from geometry file.
+
+        Geometry geo;
+        geo.read(argv[2], argv[3], OLD_ORDERING);
+
+        // Reading the file containing the positions of the EEG patches
+
+        Sensors electrodes(argv[4]);
+
+        // Assembling Matrix from discretization:
+        // Head2ECoG is the linear application which maps x |----> v
+
+        std::cerr << "Warning: we assume that ECoG electrodes are placed on the inner interface." << std::endl
+                  << "This is only valid for nested files. Consider specifying an interface as a name" << std::endl
+                  << " right after the electrode position file." << std::endl;
+        const Interface& innerSkullLayer = geo.innermost_interface();
+        std::cerr << "Detected interface: " << innerSkullLayer.name() << ' ' << geo.interface("3").name() << std::endl;
+        Head2ECoGMat mat(geo,electrodes,innerSkullLayer);
```

Comment:
why is it std::cerr and not cout ?
why is it the interface "3" ? interfaces can be named in the geom file.